### PR TITLE
test: increase e2e test run with 15 minutes

### PR DESCRIPTION
**Pull request by sinnykumari**
fixes e2e-gcp-op test failing in ci due to timeout

### CODE REVIEW

**Comment by sinnykumari** [comment]
By analyzing some of gcp-op test run logs, it seems system reboot time has increased by around 30 seconds.

1. With increased reboot time ~90 seconds
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_machine-config-operator/2181/pull-ci-openshift-machine-config-operator-master-e2e-gcp-op/1321047711511744512/artifacts/e2e-gcp-op/pods/openshift-machine-config-operator_machine-config-daemon-ggxs5_machine-config-daemon.log
```
I1027 12:52:45.162142    1921 update.go:1607] initiating reboot: Node will reboot into config rendered-infra-d181f646f702e6ac241a4123a9ab0767
I1027 12:52:45.274952    1921 daemon.go:641] Shutting down MachineConfigDaemon
I1027 12:54:18.222134    2081 start.go:108] Version: machine-config-daemon-4.6.0-202006240615.p0-370-gf72a5ace-dirty (f72a5ace0b5432ad96bba59b2b91633d8bb8315c)
I1027 12:54:18.227850    2081 start.go:121] Calling chroot("/rootfs")
I1027 12:54:18.228019    2081 rpm-ostree.go:261] Running captured: rpm-ostree status --json
I1027 12:54:18.599291    2081 daemon.go:226] Booted osImageURL: registry.build01.ci.openshift.org/ci-op-gdsvbywh/stable@sha256:cd6d0d6c4cbaa7ceaca75e7a00fad2ced6344bc89c38cadfa121b24209038e2f (47.82.202010270142-0)/stable@sha256:cd6d0d6c4cbaa7ceaca75e7a00fad2ced6344bc89c38cadfa121b24209038e2f (47.82.202010270142-0)
```

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_machine-config-operator/2181/pull-ci-openshift-machine-config-operator-master-e2e-gcp-op/1321047711511744512/artifacts/e2e-gcp-op/pods/openshift-machine-config-operator_machine-config-daemon-h5bd9_machine-config-daemon.log

```
I1027 12:09:08.575381    1848 update.go:1607] initiating reboot: Node will reboot into config rendered-worker-4ad27ca30531f77416a00e38bda8c8e6
I1027 12:09:08.668785    1848 daemon.go:641] Shutting down MachineConfigDaemon
I1027 12:10:44.764737    2142 start.go:108] Version: machine-config-daemon-4.6.0-202006240615.p0-370-gf72a5ace-dirty (f72a5ace0b5432ad96bba59b2b91633d8bb8315c)
I1027 12:10:44.772268    2142 start.go:121] Calling chroot("/rootfs")
I1027 12:10:44.772468    2142 rpm-ostree.go:261] Running captured: rpm-ostree status --json
I1027 12:10:45.343467    2142 daemon.go:226] Booted osImageURL: registry.build01.ci.openshift.org/ci-op-gdsvbywh/stable@sha256:cd6d0d6c4cbaa7ceaca75e7a00fad2ced6344bc89c38cadfa121b24209038e2f (47.82.202010270142-0)
```

2. Previous reboot time ~60 seconds

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_machine-config-operator/2177/pull-ci-openshift-machine-config-operator-master-e2e-gcp-op/1319724839077941248/artifacts/e2e-gcp-op/pods/openshift-machine-config-operator_machine-config-daemon-wmvdr_machine-config-daemon.log

```
I1023 21:06:25.466932    1842 update.go:1590] initiating reboot: Node will reboot into config rendered-infra-54a8f08a03796f8c94187a986edcbc7a
I1023 21:06:25.594958    1842 daemon.go:641] Shutting down MachineConfigDaemon
I1023 21:07:30.693671    1850 start.go:108] Version: machine-config-daemon-4.6.0-202006240615.p0-370-g56ded555-dirty (56ded5550c030d88cdffa8de630ff1a1287303f3)
I1023 21:07:30.700093    1850 start.go:121] Calling chroot("/rootfs")
I1023 21:07:30.700507    1850 rpm-ostree.go:261] Running captured: rpm-ostree status --json
I1023 21:07:31.107832    1850 daemon.go:226] Booted osImageURL: registry.build01.ci.openshift.org/ci-op-vcw3wmdq/stable@sha256:113a36da35d4aff5f8ef43ff97bed0e97cdaea2139c8db44fbac31051bec43c8 (47.82.202010231442-0)
```

**Comment by kikisdeliveryservice** [comment]
I think we should figure out the underlying problem and not extend the test time.

**Comment by kikisdeliveryservice** [comment]
(As per slack, we're doing some investigation on this before deciding how to resolve)

**Comment by cgwalters** [comment]
Sadly we never summarized this bug anywhere public on the MCO repo AFAICS.

https://github.com/openshift/cluster-network-operator/pull/859 merged which should help - let's try to verify that.
The other case is https://bugzilla.redhat.com/show_bug.cgi?id=1893360 - it needs some design work.

**Comment by cgwalters** [comment]
OK another status update on this; I kept being confused how it wasn't helping but it turns out CI (and nightly) payload generation has been broken until *just now*, so our CI runs were still using an old cluster-network-operator.

Let's keep an eye out now to see if https://github.com/openshift/cluster-network-operator/pull/859 actually improves things!

**Comment by sinnykumari** [comment]
nightly images are green now, https://github.com/openshift/cluster-network-operator/pull/859 should be included in recent payload in ci run now. For sanity check, re triggered test here.

**Comment by cgwalters** [comment]
We should probably consider this to start clearing out the PR backlog.
/retest

**Comment by kikisdeliveryservice** [comment]
> We should probably consider this to start clearing out the PR backlog.

e2e-aws and e2e-aws-serial are also both currently broken and being worked on. we shouldnt be overriding all of those (3+) tests just to get prs in imo. 

for gcp-op: https://github.com/openshift/cluster-dns-operator/pull/213#issuecomment-728219825  (and https://github.com/openshift/machine-config-operator/pull/2229) need to be merged but can't bc of the above e2e-aws issues

so it seems like first the e2e-aws tests needs to get fixed bc that's blocking the dns pr which once that merges unblocks our ci.

**Comment by cgwalters** [comment]
It's all interlinked though.  We now have so many PRs outstanding that we're waiting on AWS "leases" in some, e.g.
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-dns-operator/213/pull-ci-openshift-cluster-dns-operator-master-e2e-aws/1328393945482268672
is waiting - it won't even start until we clear the queue some.

For example we could combine this PR with https://github.com/openshift/machine-config-operator/pull/2229
and merge when e2e-gcp-op goes green, even if e2e-aws isn't (changes in our tests clearly can't affect that).

Basically I think we should try to do *something* other than be blocked.

**Comment by kikisdeliveryservice** [comment]
Right but a bigger problem mentioned in slack is that ci and nightly payload acceptance is also broken due to this...

I'm going to go and try to create more urgency on the aws bc we shouldnt be merging in these condiation but it should also be more important than it is rn.

**Comment by kikisdeliveryservice** [comment]
As an update: this comes down to https://bugzilla.redhat.com/show_bug.cgi?id=1897604 (which is also blocking the dns fix: https://github.com/openshift/cluster-dns-operator/pull/213) and blocking all merges across ocp.  There's a new channel (incident-kcm..) now where people having started working on it.

Instead of trying to hack around and merge this, which won't help bc other required tests are blocking on all repos incl this one, we are waiting for the above bz to be resolved.

**Comment by sinnykumari** [comment]
Closing this PR since actual slowness issue has been fixed with https://github.com/openshift/cluster-dns-operator/pull/213
/close
